bluesky/callbacks/mpl_plotting.py

```diff
@@ -468,6 +476,50 @@ def stop(self, doc):
         # Intentionally override LivePlot.stop. Do not call super().
 
 
+class LiveWaterfall(CallbackBase):
```

Comment:
Is there any other name for this than 'waterfall'?  There is also a community that calls khymographs 'waterfall' plots so there is not universal agreement on the meaning.

How about `OffsetLines`, `ShiftedLines`, or `StackedLines` (`stackplot` is already a thing https://matplotlib.org/api/_as_gen/matplotlib.axes.Axes.stackplot.html) ?